File: ARCADE/Intro/Level 1_2: centuryFromYear.cs
    using System;
     
    public class Test
    {
    	public static int centuryFromYear(int year){
            int m = (year % 100);
            if (m > 0) return (year / 100 + 1);
            return (year / 100);
    	}
    	public static void Main()
    	{
    		int year = 1905;
    		Console.WriteLine(centuryFromYear(year));
    		int year1 = 1700;
    		Console.WriteLine(centuryFromYear(year1));
    	}
    }

File: ARCADE/Intro/Level 1_2: centuryFromYear.pas
    program centuryFromYear;
     
    function centuryFromYear(year: integer): integer;
    var m : integer;
    begin
        m := year mod 100;
        if(m > 0) then centuryFromYear := year div 100 + 1
        else centuryFromYear := year div 100;
    end;
     
    var year : integer;
    begin
    	year := 1905;
    	writeln(centuryFromYear(year));
    	year := 1700;
    	writeln(centuryFromYear(year));
    end.
